homeassistant/bootstrap.py

```diff
@@ -216,40 +223,51 @@ def async_from_config_file(config_path: str,
 
 
 @core.callback
-def async_enable_logging(hass: core.HomeAssistant, verbose: bool = False,
-                         log_rotate_days=None, log_file=None) -> None:
+def async_enable_logging(hass: core.HomeAssistant,
+                         verbose: bool = False,
+                         log_rotate_days=None,
+                         log_file=None,
+                         log_no_color: bool = True) -> None:
```

Comment:
This should not default to True

Reply:
Whoops, I initially implemented this as log_color, but then switched it around to make using the new switch statement more clean.  I'll switch all these to False.